### PR TITLE
Walls upd

### PR DESCRIPTION
**Pull request by Jaraten**

### CODE REVIEW

File: Resources/Prototypes/_CP14/Entities/Structures/Walls/walls.yml [hunk 1]
```diff
@@ -116,4 +116,16 @@
         acts: [ "Destruction" ]
   - type: Construction
     graph: CP14WallWood
-    node: WallCardboard
\ No newline at end of file
+    node: WallCardboard
+    
+- type: entity
+  id: CP14WallCyan
+  name: cyan wall
```
Comment (TheShuEd) [inline Resources/Prototypes/_CP14/Entities/Structures/Walls/walls.yml:123]: maybe smtn like polished "marble" or "granite"

Comment (Jaraten) [reply Resources/Prototypes/_CP14/Entities/Structures/Walls/walls.yml:123]: Later, there will be separate granite walls

Comment (TheShuEd) [reply Resources/Prototypes/_CP14/Entities/Structures/Walls/walls.yml:123]: Well, anyway, just the cyan wall is a weird name.

Comment (Jaraten) [reply Resources/Prototypes/_CP14/Entities/Structures/Walls/walls.yml:123]: It's more like spackle

Comment (Jaraten) [reply Resources/Prototypes/_CP14/Entities/Structures/Walls/walls.yml:123]: I have a bad fantasy(

**Pull request merged**